selfdrive/car/fw_versions.py

```diff
@@ -97,6 +97,11 @@ def p16(val):
 CHRYSLER_VERSION_RESPONSE = bytes([uds.SERVICE_TYPE.READ_DATA_BY_IDENTIFIER + 0x40]) + \
   p16(0xf132)
 
+CHRYSLER_VERSION_REQUEST_ALT = bytes([uds.SERVICE_TYPE.READ_DATA_BY_IDENTIFIER]) + \
+  p16(uds.DATA_IDENTIFIER_TYPE.SYSTEM_SUPPLIER_ECU_SOFTWARE_NUMBER)
+CHRYSLER_VERSION_RESPONSE_ALT = bytes([uds.SERVICE_TYPE.READ_DATA_BY_IDENTIFIER + 0x40]) + \
+  p16(uds.DATA_IDENTIFIER_TYPE.SYSTEM_SUPPLIER_ECU_SOFTWARE_NUMBER)
+
```

Comment:
@realfast have any clue which of the two identifiers that look most correct, is correct?

![Screenshot from 2022-08-20 00-18-59](https://user-images.githubusercontent.com/25857203/185733954-87835067-2f4b-4810-92b2-9fcbc03ec7a2.png)

Reply:
Should use software, as hardware can be the same across different EPS's despite different software capabilities